node-runner-cli/utils/utils.py

```diff
@@ -33,6 +33,25 @@ def run_shell_command(cmd, env=None, shell=False, fail_on_error=True, quite=Fals
         sys.exit()
     return result
 
+def print_vote_and_fork_info(health, engine_configuration):
+    vote_status = health['fork_vote_status']
+    print(f"Vote status : {vote_status}")
+    if vote_status == 'VOTE_REQUIRED':
+        print(f"{bcolors.WARNING}Your vote is required{bcolors.ENDC}")    
+    else:
+        print(f"{bcolors.WARNING}No vote is required at the moment{bcolors.ENDC}")    
+    latest_fork = engine_configuration['forks'][-1] 
+    latest_fork_name = latest_fork['name']
+    is_candidate = latest_fork['is_candidate']
+    if health['current_fork_name'] == latest_fork_name:        
+        print(f"\nCurrent fork : {latest_fork_name}, is candidate: {is_candidate}")    
+        print(f"{bcolors.WARNING}The validator is on the latest fork{bcolors.ENDC}")
```

Comment:
This is probably more accurately "The validator is currently running its newest fork"